docs/release_notes/v1.12.1.md

```diff
@@ -22,3 +52,134 @@ Sentry also still accepts requests which use the legacy identifier of `<namespac
+
+### Problem
+
+Invoking certain Workflow APIs using HTTP with malformed URLs can cause Dapr to panic.
+
+### Impact
+
+Impacts users on Dapr 1.12.0.
+
+### Root cause
+
+The Daprd metrics handler for workflows did not correctly handle malformed or unexpected URLs, and would panic if the URL was not in the expected format.
+
+### Solution
+
+The Daprd metrics handler for workflows now correctly handles malformed or unexpected URLs.
+
+
+## Fixed an issue where Azure Blob Storage components cannot be used with Azure Blob Reader permission alone
+
+### Problem
+
+The Azure Blob Storage component cannot be used with Azure Blob Reader permission alone.
+
+### Impact
+
+Users on Dapr 1.12.0 cannot use the Azure Blob Storage component with Azure Blob Reader permission alone.
+
+### Root Cause
+
+The component attempts to create a storage account (even when it exists already) but is not authorized to do so due to a lack of permission.
+
+### Solution
+
+Added a boolean metadata option disableEntityManagement which now can be used to skip the attempt to create the storage container.
+
+## Fixed incorrect error message in the Azure EventHubs components
+
+### Problem
+
+Azure EventHub component returns warning message says the storageAccountKey will be used but StorageAccountName is actually used in that condition.
+
+### Impact
+
+Users on Dapr 1.12.0 may be confused by the warning message.
+
+### Root Cause
+
+The component was using the StorageAccountName instead of the StorageAccountKey.
+
+### Solution
+
+The component now emits the correct warning message.
+
+## Fixes an issue where the Consul nameresolution component only accepted IP addresses and not hostnames.
+
+### Problem
+
+Dapr 1.12.0 was unable to add and resolve hostnames using the Consul nameresolution component.
+
+### Impact
+
+Users on Dapr 1.12.0 were unable to resolve hostnames using the Consul nameresolution component.
+
+### Root Cause
+
+A previous community contribution to improve IPv6 support inadvertently removed support for host names by returning an error for any string that does not parse as an IPv4 or IPv6 address.
+
+### Solution
+
+Support for hostnames was added once again.
+
+## Fixes an issue in the Redis PubSub component where a PubSub subscription could not recover under certain conditions.
+
+### Problem
+
+The Redis PubSub component could experience a rare situation where PubSub subscriptions would seize to function, causing the Dapr sidecar to print Redis error logs containing the word `NOGROUP` indefinitely.
+
+### Root Cause
+
+The `NOGROUP` message is indicative of the consumer group no longer existing. This could be because the group or stream were manually deleted outside of Dapr or because the external Redis server was restarted/reconfigured.
+
+### Solution
+
+To avoid the excessive `NOGROUP` logs in this rare situation Dapr will now attempt to recreate the consumer group for the given topic / stream when this error is encountered.
```

Comment:
```suggestion
To avoid the excessive `NOGROUP` logs in this rare situation Dapr will now attempt to recreate the consumer group for the given topic/stream when this error is encountered.
```